build/build/src/engine/bundle.rs

```diff
@@ -7,6 +7,13 @@ use ide_ci::cache::goodie::graalvm::locate_graal;
 
 
 
+/// Version of the bundled GraalVM.
+#[derive(Clone, Debug)]
+pub struct GraalVmVersion {
+    pub graal:    Version,
+    pub packages: Version,
```

Comment:
Some comment explaining what these versions actually are and why they are different would be nice.